operatorhub/kubernetes/release-artifacts/bundle/manifests/operator.tekton.dev_tektonconfigs.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
so we have all crds in `release-artifacts` are they manually added?
and do we need to keep them here or are they copied while creating csv?

Reply:
as part of the bundle generation, CRDs, configmaps etc in the accumulated list (from local or operator release.yaml) ends up as seprate files beside the ClusterSerciviceVersion file.